contracts/Org.sol

```diff
@@ -18,23 +18,27 @@ import "@openzeppelin/contracts/token/ERC20/SafeERC20.sol";
  */
 contract Org is Administratable {
   using SafeERC20 for IERC20;
-  // ========== STATE VARIABLES ==========
+
+  // ========== STRUCTS & EVENTS ==========
 
   struct Claim {
     string firstName;
     string lastName;
     string eMail;
     address desiredWallet;
-    bool filesSubmitted;
   }
 
+  event CashOutComplete(uint256 cashOutAmount);
+  event ClaimCreated(string claimId, Claim claim);
+  event ClaimApproved(string claimId, Claim claim);
+  event ClaimRejected(string claimId, Claim claim);
+
+  // ========== STATE VARIABLES ==========
+
   uint256 public taxId;
-  address public orgWallet;
-  Claim[] public claims;
 
-  event CashOutComplete(uint256 cashOutAmount);
-  event ClaimCreated(Claim claim);
-  event ClaimApproved(Claim claim);
+  mapping(string => Claim) public pendingClaims; // claim UUID to Claim
```

Comment:
Could reduce gas costs in a lot of places by instead using a mapping from `bytes32 => Claim` where `bytes32` is the hash of the UUID

Reply:
+1 good shout

Reply:
If we did this, would we want to pass the hash in to the `requestClaim`, `acceptClaim`, and `rejectClaim` method, or the string itself?

If the latter, does calculating the hash and storing it still work out to less gas than storing a UUID sized string? (I'm guessing yes, but would want to confirm).

If the former, we won't be able to announce the string version of the UUID in the emitted event. This means, to know the UUID of existing claims, you'd have to get them from somewhere else, namely the database, or the endaoment API if you're a third party. Perhaps that's fine, just want to point out the tradeoff!

Reply:
I was thinking we'd pass the hash in directly, but good point about events. We could just store the hash and UUID in the DB like you suggested, but I'm not totally sure who will be needing it, so will defer to @rheeger on that

Reply:
A few questions: How much of a savings are we talking about? Won't a hash reduce the integrity (however marginally) of the UUID? I wonder cause users will want to see the uuid if they inspect the blockchain. @apbendi We can also calc hash off chain.

Reply:
A quick storage test shows 46080 gas to save a string to a mapping, and 44599 gas to save the keccak256 hash of that string to a mapping. So it seems because the string is relatively short it doesn't make a big difference, so might be easiest to leave this as a string. Contract is below if anyone wants to reproduce:

```solidity
pragma solidity ^0.6.7;

contract StringTest {
     mapping (string => bool) public stringCheck;
     mapping (bytes32 => bool) public bytesCheck;
     
    function saveString(string memory _var) public {
        stringCheck[_var] = true;
    }

     function saveBytes(bytes32 _var) public {
        bytesCheck[_var] = true;
    }
}
```